core/core.go

```diff
@@ -250,19 +250,30 @@ func findWorkspaceRoot(root string) string {
 	return findWorkspaceRoot(parentDirectory)
 }
 
+// TODO(go 1.18): remove backport of strings.Cut
+func strings_Cut(s, sep string) (before, after string, found bool) {
```

Comment:
`Cut` is simpler.

Reply:
I'm still in favor of Split since it would avoid the custom function for now. If you want to keep this function, then please give it a name that is consistent with the others (otherwise our linter will complain), e.g. cutString